### ISSUE TITLE
Feature that allows axis inside the plot

### ISSUE THREAD

**Issue by 38** [38/plotters#33]
See discuss under issue #31

**Comment by serzhiio** [comment]
Any idea how to inset chart axis and labels while its not implemented?

**Comment by 38** [comment]
The fastest way, you may already know is draw it yourself. Since Plotters expose full drawing API to developer. 

Things may be helpful to you is:

First, we can use this https://docs.rs/plotters/0.2.7/plotters/chart/struct.ChartContext.html#method.plotting_area To get the plotting area.

Then, you can use https://docs.rs/plotters/0.2.7/plotters/drawing/struct.DrawingArea.html#method.draw to draw anything you want on to it. 

So the thing may looks like:

```rust
      let mut chart ....;
      let pda = chart.plotting_area().strip_coord_sepc();
      let range: RangedCoordi32 = (0..100).into();
      for y in range.key_points(5) {
                // In here we get desired location for the tick marks. 
                pda.draw(.....);
      }
```

Hopefully you can draw anything you want. Feel free to ask me if you have any question :)

**Comment by serzhiio** [comment]
I have almost done this feature.
Hope will PR soon...

**Comment by 38** [comment]
Merged #50
